PennMobile/src/main/java/com/pennapps/labs/pennmobile/DirectoryFragment.java

```diff
@@ -32,15 +42,15 @@
     private ListView mListView;
     private Context mContext;
     private SearchView searchView;
+    private boolean starOn;
+    private DirectoryAdapter mainAdapter;
+    private Menu mMenu;
+    private int state;
+    private final int NO_STATE = 1, INSTR_STATE = 0, LIST_STATE = 2;
```

Comment:
Use https://developer.android.com/reference/android/support/annotation/IntDef.html to give some extra guarantees.